build.gradle

```diff
@@ -26,7 +26,7 @@ def dependencyVersions = [
         ida_utils             : '2.0.0-395',
         dropwizard            : '1.3.8',
         verify_dev_pki        : '2.0.0-45',
-        saml_libs             : "$opensaml_version-254",
+        saml_libs             : "$opensaml_version-258",
         ida_test_utils        : '2.0.0-65',
         hub_saml              : "$opensaml_version-15825",
```

Comment:
Is this going to break because you've changed `opensaml_version`? Looks like it's pinned to a very specific build?